tests/test_13_validate.py

```diff
@@ -33,7 +33,7 @@ def test_duration():
     assert valid_duration("-P1347M")
     assert valid_duration("P1Y2MT2.5H")
 
-    raises(NotValid, 'valid_duration("P-1347M")')
+    # XXX raises(NotValid, 'valid_duration("P-1347M")')
```

Comment:
That is right; however, `aniso8601` does support it and thus we do too. This will change soon, as `aniso8601` wants to strictly conform to ISO 8601 and negative durations will be dropped. You can read about this in [issue#20][nd]. Until then, I am planning to change this comment to [an expected failure][xf]. 

  [nd]: https://bitbucket.org/nielsenb/aniso8601/issues/20/negative-duration
  [xf]: https://docs.pytest.org/en/latest/skipping.html#xfail-mark-test-functions-as-expected-to-fail

Reply:
Cool, sounds like good decisions all around.